Modules/Relationships/src/Relationships.Application/Relationships/Commands/RelationshipReactivationRequest/RelationshipReactivationRequestCommand.cs

```diff
@@ -0,0 +1,7 @@
+﻿using MediatR;
+
+namespace Backbone.Modules.Relationships.Application.Relationships.Commands.RelationshipReactivationRequest;
+public class RelationshipReactivationRequestCommand : IRequest<RelationshipReactivationRequestResponse>
```

Comment:
Currently you don't enforce the quotas on NumberOfRelationships

Reply:
Changes made.